src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -456,7 +457,7 @@ def getClusterAd(self):
         ad['Cmd'] = self.scriptFile
         ad['Out'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".out")')
         ad['Err'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".err")')
-        ad['UserLog'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".log")')
+        ad['UserLog'] = classad.ExprTree('strcat(Iwd, "/condor.", ClusterId, ".", ProcId, ".log")')
```

Comment:
@amaltaro , I had it for none of them and that worked as far as I remembered. Why did you add it for the UserLog ? Did you see problems with it ?